charts/jaeger/values.yaml

```diff
@@ -431,7 +440,7 @@ spark:
   podLabels: {}
 
 esIndexCleaner:
-  enabled: false
+  enabled: true
```

Comment:
Just curious, why are we making this true by default ?
Also, if we are doing  so, are we adding it to notes or README ? (for some people this might be a breaking change if they were not expecting to loose data from es-index-cleaner cron job (as it was false before)

Reply:
This is just me being sloppy, good catch!